src/main/java/edu/harvard/iq/dataverse/DataCitation.java

```diff
@@ -269,9 +269,17 @@ public void writeAsBibtexCitation(OutputStream os) throws IOException {
             out.write(title);
             out.write("},\r\n");
         }
+        if(getUNF() != null){
+            out.write("UNF = {");
+            out.write(UNF);
```

Comment:
Is there a reason to use `getUNF()` and `UNF` in the same block?

Reply:
Not really, just following the pattern wrt title above.  I can change it.